### ISSUE TITLE
CVAT doesn't export instance annotations in the correct format

### ISSUE THREAD

**Issue by davodogster** [openvinotoolkit/cvat#2223]
Hi there. I'm a big fan of using the openvino model assisted annotation. So I need to train a MaskRCNN and convert it to openvino then upload the model to cvat (have done it for faster rcnn). But to train MaskRCNN you need a TFrecord file and for that you need the XMLS( in the correct format) with the bounding box coordinates of the polygons and also the PNG masks. When I dump annotations in the sementation mask format I don't get the XMLS. When I export in other format coco it has a json file which has the bbox co-ords but that is not what's needed.. We need an xml per image not one big Json file in the wrong format.

CVAT doesn't export the write annotation files needed to (convert to tfrecord) and train maskRCNN.

I'm Running CVAT in docker. Using a slightly older version - just before serverless functions were introduced. 

Cheers,
Sam

**Comment by zhiltsov-max** [comment]
Hi, have you tried exporting as `PASCAL VOC`?

**Comment by davodogster** [comment]
Yes and since it’s polygon annotations, when I export as pascal VOC it Doesn’t output any annotations. Just empty folders
________________________________

**Comment by zhiltsov-max** [comment]
Ok, makes sense. What about `TFRecord` format?

**Comment by davodogster** [comment]
I dont think the TFRECORD export will work. I exported Bbox tfrecord once and they didn’t work. Even if it does.. I need to split it into train,Val, test which is impossible.

________________________________

**Comment by davodogster** [comment]
@zhiltsov-max it turns out that exporting Polygon annotations as COCO and using create_coco_tf_record.py with the instances_default.json file, include_masks=True, from CVAT enables us to train a TF1OD mask RCNN (I believe).
